docs/kb/api-version-4.md

```diff
@@ -1,48 +1,98 @@
 We are now allowing multiple prospects to share an email address on some Pardot accounts.
 If your account has this feature active now, then you MUST use version 4.
 Everyone else can continue using version 3.
 
+For help determining if you have access to version 4, check for the "Enabled" status following [this guide.](http://help.pardot.com/customer/portal/articles/2461386-how-can-i-find-out-if-my-account-allows-multiple-prospects-with-the-same-email-address-)
+
 For more information on the API in general, check out the [Overview Page](/).
 
 For specific API endpoints, please select from the "Version 4" list above.
 
-## Key Differences
-
-Version 4 may use slightly different input syntax where prospects are involved,
-and may return multiple prospects where version 3 returned one.
-Please check out the appropriate version's documentation for more exact usage details.
 
-If your account uses version 4 of the API, then you must include the version 4 tag in your requests.
-Failure to do so will result in an error about the API being requested.
+# Transitioning from version 3 to version 4
 
-The biggest change in the APIs in this version are that we now allow for multiple prospect records with the same email address.
-A few examples of what that means:
+If you are looking to upgrade your account to 
+[allow multiple prospects with the same email address](http://help.pardot.com/customer/portal/articles/2461386-how-can-i-find-out-if-my-account-allows-multiple-prospects-with-the-same-email-address-),
+and you use the api, you will need to follow this guide.
 
-* Prospect Create by email will create a new prospect for each time the call is made.
-* Prospect Upsert by email will also always create under this specification.
-* Prospect Update by email will no longer work (we don't necessarily know which prospect you're talking about)
+## Determining when these changes are applicable
 
-### Requesting an API key
-
-An exception to this rule is requesting an `api_key` through the login API.
-You may use `/version/3` or `/version/4` for this call.
-To let you know that you must use version 4, we will return a `<version>4</version>` tag in the response.
-This version tag will not be included if your account cannot use version 4.
+When your account has access to version 4, your API session will be cleared and you must log in again.
+Upon logging in, a new field will be returned in the login response. 
+This will be your indication that you may no longer use version 3 (or any other previous version), and must now use the version 4 syntax.
+Here's what that will look like:
 
 Request:
 ```
 POST: https://pi.pardot.com/api/login/version/3
 message body: email=<email>&password=<password>&user_key=<user_key>
 ```
 
-Response:
+Response (Before transition):
+```
+<rsp stat="ok" version="1.0">
+  <api_key>1234abcd</api_key>
+</rsp>
+```
+
+Response (After transition):
 ```
 <rsp stat="ok" version="1.0">
   <api_key>1234abcd</api_key>
```

Comment:
Might want to use a different example for the key here to highlight that the key won't be the same from pre-transition to post-transition.